src/main/java/uk/gov/hmcts/reform/em/stitching/rest/WelcomeResource.java

```diff
@@ -31,13 +33,24 @@ public class WelcomeResource {
         produces = APPLICATION_JSON_VALUE
     )
     @ResponseBody
-    public ResponseEntity<String> welcome() {
+    public ResponseEntity<Object> welcome() {
 
         log.info("Welcome message : '{}'", MESSAGE);
 
+        Map<String, Object> data = new HashMap<>();
+        data.put("message", MESSAGE);
+
+        ObjectMapper mapper = new ObjectMapper();
+        String jsonString;
+
+        try {
+            jsonString = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(data);
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
```

Comment:
do not need this,  you can directly return Map<String,String> or create POJO class for welcome Object with field `message`

Reply:
Okay I will change that